netlify.toml

```diff
@@ -34,3 +34,7 @@ ID = "b0838013-ab4a-4e8d-a266-0eb61405ad77"
 # inherit these settings.
 # [context.deploy-preview]
   # publish = "dist/"
+
+[[redirects]]
+  from = "/:version/features/telemetry"
+  to = "/:version/features/correlation#logging"
```

Comment:
Great that you though of this.

Reply:
Thx! It's a bit tricky though, bc we can't remove the `# Logging` header, otherwise we again have death links 😄 .

Reply:
But I'd assume we do not need to keep this redirect around for eternity ?

Reply:
Not sure, 🤷‍♂️.